src/Workspaces/Core/Portable/Storage/AbstractPersistentStorageService.cs

```diff
@@ -95,7 +95,7 @@ public ValueTask<IChecksummedPersistentStorage> GetStorageAsync(Workspace worksp
                     // This will remove the single ref count we ourselves added when we cached the
                     // instance.  Then once all other existing clients who are holding onto this
                     // instance let go, it will finally get truly disposed.
-                    _ = Task.Run(() => storageToDispose.Dispose());
+                    _ = Task.Run(() => storageToDispose.Dispose(), CancellationToken.None);
```

Comment:
Can you add a comment that this is absolutely not cancellable and the disposal must happen? thanks